src/test/kotlin/no/nav/helsearbeidsgiver/aareg/AaregClientTest.kt

```diff
@@ -1,41 +1,45 @@
 package no.nav.helsearbeidsgiver.aareg
 
+import io.kotest.assertions.throwables.shouldThrowExactly
 import io.kotest.core.spec.style.FunSpec
-import io.kotest.matchers.collections.shouldBeEmpty
-import io.kotest.matchers.collections.shouldNotBeEmpty
+import io.kotest.matchers.maps.shouldContainExactly
+import io.ktor.client.plugins.ServerResponseException
 import io.ktor.http.HttpStatusCode
+import io.ktor.serialization.JsonConvertException
+import no.nav.helsearbeidsgiver.utils.test.date.januar
+import no.nav.helsearbeidsgiver.utils.test.date.mars
+import no.nav.helsearbeidsgiver.utils.test.wrapper.genererGyldig
+import no.nav.helsearbeidsgiver.utils.wrapper.Fnr
+import no.nav.helsearbeidsgiver.utils.wrapper.Orgnr
 
 class AaregClientTest : FunSpec({
 
-    /*
-    API Description:
-    https://navikt.github.io/aareg/tjenester/integrasjon/api/
-     */
     test("gir ikke-tom liste med arbeidsforhold") {
         val response = mockAaregClient(MockResponse.arbeidsforhold)
-            .hentArbeidsforhold("ident", "call-id")
+            .hentAnsettelsesperioder(Fnr("22018520056"), "call-id")
```

Comment:
Sikkert ikke ekte Fnr, men kanskje bruke noe ala Fnr.genererGyldig() el.l, for å unngå "mistenkelige" / "sensitiv-smelling" variabler i koden

Reply:
Dette fnr-et ligger i json-en som blir returnert (`MockResponse.arbeidsforhold`), så tenkte jeg at det var best at kallet matchet responsen.